pkg/controller/pod.go

```diff
@@ -131,6 +139,39 @@ func (pnc *PodNetworksController) worker() {
 	}
 }
 
+func (pnc *PodNetworksController) ignoreHostNetworkedPods(pod *corev1.Pod) {
+	klog.Warningf(
+		"rejecting to add interfaces for host networked pod: %s",
+		annotations.NamespacedName(pod.GetNamespace(), pod.GetName()),
+	)
+	pnc.Eventf(pod, corev1.EventTypeWarning, "InterfaceAddRejected", rejectInterfaceAddEventFormat(pod))
+}
+
+func (pnc *PodNetworksController) reconcileOnStartup() error {
+	pods, err := pnc.podsLister.List(labels.Everything())
+	if err != nil {
+		err = fmt.Errorf("failed to list pods on current node: %v", err)
+		return err
```

Comment:
nit: can we just inline these ? 
```suggestion
		return fmt.Errorf("failed to list pods on current node: %v", err)
```